plugins/discourse-narrative-bot/plugin.rb

```diff
@@ -48,12 +48,16 @@
     '../lib/discourse_narrative_bot/welcome_post_type_site_setting.rb'
   ].each { |path| load File.expand_path(path, __FILE__) }
 
-  # Disable welcome message because that is what the bot is supposed to replace.
-  SiteSetting.send_welcome_message = false if SiteSetting.send_welcome_message
-
-  certificate_path = "#{Discourse.base_url}/discobot/certificate.svg"
-  if SiteSetting.discourse_narrative_bot_enabled && !SiteSetting.allowed_iframes.include?(certificate_path)
-    SiteSetting.allowed_iframes = SiteSetting.allowed_iframes.split('|').append("#{Discourse.base_url}/discobot/certificate.svg").join('|')
+  RailsMultisite::ConnectionManagement.each_connection do
```

Comment:
Is that the only piece of code we do that needs to be done properly for multisite?
Did you check if there was an helper in the plugin API?

Reply:
Yes, it is. We do not have the tools to automatically test this, but locally this is all it is needed.